css/modules/aioseop_module.css

```diff
@@ -1,1120 +1,1350 @@
+	margin-bottom: 5px;
+}
+
+#aiosp_feature_manager_metabox.postbox {
+	margin-top: 20px;
+	float: left;
+}
+
+.aioseop_advert p {
+	margin: 25px 0 25px 0;
+}
+
+/*
+.widget, #widget-list .widget-top, .postbox, #titlediv, .aioseop_options_wrapper .postarea, .stuffbox {
+	border-color: #DFDFDF;
+	-moz-box-shadow: inset 0 1px 0 #fff;
+	-webkit-box-shadow: inset 0 1px 0 #fff;
+	box-shadow: inset 0 1px 0 #fff;
+	-moz-border-radius: 3px;
+	-khtml-border-radius: 3px;
+	-webkit-border-radius: 3px;
+	border-radius: 3px;
+}
+*/
+
+.aioseop_options_wrapper .postarea {
+	border-color: #DFDFDF;
+	-moz-box-shadow: inset 0 1px 0 #fff;
+	-webkit-box-shadow: inset 0 1px 0 #fff;
+	box-shadow: inset 0 1px 0 #fff;
+	-moz-border-radius: 3px;
+	-khtml-border-radius: 3px;
+	-webkit-border-radius: 3px;
+	border-radius: 3px;
+}
+
+.aioseop_advert h3 {
+	padding: 0;
+	margin-top: 6px;
+}
+
+.aioseop_metabox_text p {
+	margin: 0 0 0px 0;
+	width: 101%;
+}
+
+.aioseop_sidebar {
+	width: 457px;
+	margin-left: 10px;
+}
+
+.aioseop_metabox_text {
+	margin-bottom: 0px;
+}
+
+.aioseop_metabox_wrapper {
+	padding: 0px;
+}
+
+.aioseop_metabox_text *:last-child {
+	margin: 0;
+}
+
+.aioseop_metabox_feature {
+	margin-top: 20px;
+}
+
+.aioseop_metabox_feature img {
+}
+
+.aioseop_option_label {
+	float: left;
+	margin: 0px;
+	min-width: 150px;
+	width: 37%;
+	max-width: 270px;
+	padding-top: 3px;
+	padding-bottom: 3px;
+	height:67px !important; /*added for certain language support*/
+}
+
+.aioseop_metabox_text h2 {
+	font-size: 14px;
+	padding: 0px;
+	font-weight: bold;
+	line-height: 29px;
+}
+
+#aioseop-about {
+	width: 443px;
+	margin-bottom: 20px;
+}
+
+#aioseop-about .aioseop_metabox_text #mc-embedded-subscribe-form h2 {
+	font-size: 13px;
+}
+
+.aioseop_sidebar #mc-embedded-subscribe-form {
+	margin: 0 0 10px 0;
+	background: white;
+	padding: 10px 10px;
+	border: 1px solid #DDD;
+}
+
+.aioseop input[readonly] {
+	background-color: #EEE;
+	margin: 5px 0 5px 0 !important;
+}
+
+/*
+.aioseop input[type="button"] {
+	border-radius: 5px;
+	font-size: 15px;
+}
+*/
+
+.aioseop_settings_left {
+	float: left;
+	padding: 0px;
+	margin: 0px;
+	width: 100%;
+}
+
+body.all-in-one-seo_page_all-in-one-seo-pack-aioseop_feature_manager .aioseop_settings_left {
+	margin-top: 20px;
+}
+
+body.all-in-one-seo_page_all-in-one-seo-pack-pro-aioseop_feature_manager .aioseop_settings_left {
+	margin-top: 20px;
+}
+
+#aioseop_top_button {
+	margin-top: 5px;
+	height: 30px;
+}
+
+#aioseop-list #mce-EMAIL {
+	margin-top: 5px;
+	width: 250px;
+}
+
+.aioseop_top {
+	margin: 10px 10px 0px 0px;
+/*	margin: 10px 477px 0px 0px; */
+}
+
+.aioseop_top #aioseop-list {
+	margin-bottom: 0px;
+}
+
+.aioseop_right_sidebar {
+	float: right;
+	margin-top: 35px;
+}
+
+/*
+#wpbody-content {
+	min-width: 1180px;
+}
+*/
+
+#aiosp_settings_form .button-primary.hidden {
+	display: none;
+}
+
+form#edittag div#aiosp_titleatr_wrapper,
+form#edittag div#aiosp_menulabel_wrapper,

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Element (form#edittag) is overqualified, just use #edittag without element name.](https://www.codacy.com/app/mrtorbert/all-in-one-seo-pack/file/2692638760/issues/source?bid=3283401&fileBranchId=3340169#l505)